pkg/process/process.go

```diff
@@ -202,13 +202,30 @@ func initProcessInternalExec(
 	protoPod := GetPodInfo(containerID, process.Filename, args, process.NSPID)
 	caps := caps.GetMsgCapabilities(capabilities)
 	ns := namespace.GetMsgNamespaces(namespaces)
+	binary := path.GetBinaryAbsolutePath(process.Filename, cwd)
+
+	// Ensure that exported events have the TID set. For events from Kernel
+	// we usually use PID == 0, so instead of checking against 0, assert that
```

Comment:
Can we make this a touch clearer what we mean by "For events from Kernel we ..."->"For events generated by kernel threads we..."

Reply:
will do yes

Reply:
done thanks ;-)